cwl/main.cwl

```diff
@@ -106,7 +105,6 @@ steps:
             first_round:
               valueFrom: $(inputs.round == 0)
             input_model: input_model
-            round: round
```

Comment:
I think `round` is not listed in the `train.cwl` inputs.

Reply:
You are right, but it is present in the `first_round` `valueFrom` expression to extract if the current round is the first round or not.
The only alternative is to pass `round` to the `CommandLineTool` and to infer the value of `first_round` directly in the `arguments`. But I don't know if there is a best practice for that.

Reply:
Ah, I missed that @GlassOfWhiskey , I only checked the command line values. Dropped the commit from the PR. Sorry :+1: